Mowen_Assignment3/assignment3.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Ideally - all of these files should be in the repository for this assignment. There is a different classroom link and repository for each assignment in the course. Let me know if you have questions about this.